tests/unit/s2n_tls13_support_test.c

```diff
@@ -28,37 +28,36 @@
 int main(int argc, char **argv)
 {
     BEGIN_TEST();
-    EXPECT_SUCCESS(s2n_disable_tls13_in_test());
-
-    /* TLS 1.3 is not used by default */
-    EXPECT_FALSE(s2n_use_default_tls13_config());
+    EXPECT_SUCCESS(s2n_enable_tls13_in_test());
```

Comment:
Both s2n_disable_tls13_in_test and s2n_enable_tls13_in_test are deprecated. If you're trying to reset to the true public default behavior (which is what I assume you're trying to do here) just delete s2n_disable_tls13_in_test.